scripts/Tools/pelayout

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+"""
+This utility allows the CIME user to view and modify a case's PE layout.
```

Comment:
Yeah, I figure either take out that comment or add the modification options (see #953 for my suggestion additions).